rlbottraining/example_bots/prop_bot/prop_bot.py

```diff
@@ -0,0 +1,14 @@
+from rlbot.agents.base_agent import BaseAgent, SimpleControllerState
+from rlbot.utils.structures.game_data_struct import GameTickPacket
+
+class PropBot(BaseAgent):
+    """
+    A bot which just sits there like a prop.
```

Comment:
BrickBot already does this

Reply:
Brick bot has handbrake on, this causes the ball to move it quite a bit when it hits it.